### PR TITLE
Small install script.

### PR DESCRIPTION
**Pull request by DaveDavenport**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+DIRECTORY=${HOME}/.local/share/rofi/themes/
```
[the remark this answers is not in the document]
Comment (patrick-motard) [reply install.sh]: I was looking at the `rofi-theme-selector` script in the `rofi` project. The documentation says that it looks in 3 locations. Those three locations are neither $HOME/.local/share/rofi/themes nor $HOME/.config/rofi.
Can we add those directories to that script?

Also.. the script says you can press `alt-a` for setting a theme. I've not seen mention in the documentation anywhere on how to set a theme explicitly via configuration or a command flag. There is the deprecated `-theme` flag mentioned in `README.md`.. but that's deprecated. What is the current way of setting themes? 

I'm trying to collect as much information about this as I can so I can get it all documented in the wiki.

[the remark this answers is not in the document]
Comment (DaveDavenport) [reply install.sh]: For now `theme` is still the valid option.
In future config you can do `@import 'mytheme'`

[the remark this answers is not in the document]
Comment (patrick-motard) [reply install.sh]: Should we not mark `theme` as deprecated for now until `@import 'mytheme'` is supported? What version is that import statement going to come in?

Also, is it okay for me to document this stuff in the wiki?

[the remark this answers is not in the document]
Comment (patrick-motard) [reply install.sh]: I would be willing to submit a pr to the main rofi project to fix the deprecated part in the readme.

[the remark this answers is not in the document]
Comment (patrick-motard) [reply install.sh]: The main thrust of what I'm trying to get at here is that this install script does not place these themes in a location where rofi's theme previewer will find them.

[the remark this answers is not in the document]
Comment (DaveDavenport) [reply install.sh]: Better PR is welcome. I have little time atm to look at this.

[the remark this answers is not in the document]
Comment (carnager) [reply install.sh]: @patrick-motard 

> I was looking at the rofi-theme-selector script in the rofi project. The documentation says that > it looks in 3 locations. Those three locations are neither $HOME/.local/share/rofi/themes nor 
> $HOME/.config/rofi.

Not sure what makes you think that. Those are EXACTLY the locations mentioned in rofi-theme-selector manpage.

[the remark this answers is not in the document]
Comment (carnager) [reply install.sh]: Solution:

```
xdg=${${XDG_DATA_HOME}:-${HOME}/.local/share}
DIRECTORY="${xdg}/rofi/themes/}"
```

[the remark this answers is not in the document]
Comment (patrick-motard) [reply install.sh]: @carnager 
I just circled back to the documentation. You're right. I misread the following line:

> $XDG_CONFIG_HOME/rofi/themes

And missed:

> $XDG_CONFIG_HOME is normally unset. Default path is "$HOME/.config".

So you're totally right, my mistake!

[the remark this answers is not in the document]
Comment (patrick-motard) [reply install.sh]: Thanks again to you guys to responding to this and talking through it with me. :)

**Pull request merged**